config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,22 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// BareMetalPlatformStatus holds the current status of the BareMetal infrastructure provider.
+type BareMetalPlatformStatus struct {
+	// APIIntIP is an IP address managed by the BareMetal provider
+	// backing the internal `api-int` record.
```

Comment:
Looks like consistency went in the wrong direction.  Can you put back your better comment?  Also, we don't abbreviate `Internal`.  If this matches another status field, reference that please.